cmd/captplanet/run.go

```diff
@@ -101,11 +102,12 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 			runCfg.Satellite.Identity.Address)
 		var o provider.Responsibility = runCfg.Satellite.Overlay
 		if runCfg.Satellite.MockOverlay.Enabled {
-			o = overlay.MockConfig{Nodes: strings.Join(storagenodes, ",")}
+			fmt.Printf("STORAGE NODES = %#v\n", storagenodes)
```

Comment:
do we want to keep this print statement?

Reply:
nope :)

Reply:
done